tests/unit/suites/libraries/joomla/user/JAuthenticationTest.php

```diff
@@ -82,6 +82,7 @@ protected function tearDown()
 	{
 		// Reset the loaded plugins.
 		TestReflection::setValue('JPluginHelper', 'plugins', null);
+		$this->restoreFactoryState();
 
 		parent::tearDown();
 	}
```

Comment:
I'll do this upstream in the CMS - this might be why everything failed the other week when I was randomizing the test order (which I since have been unable to run again -.-). This PR is specifically re-enabling the tests as we had to disable them for the new dispatcher.

I agree with Michael using the built-in PHPUnit methods would be an improvement however we should do that across the CMS in one PR rather than doing it in dribs and drabs